remoteparticipant.go

```diff
@@ -29,10 +29,6 @@ func (p *RemoteParticipant) updateInfo(pi *livekit.ParticipantInfo) {
 	p.baseParticipant.updateInfo(pi, p)
 	// update tracks
 	validPubs := make(map[string]TrackPublication)
-	p.tracks.Range(func(key, value interface{}) bool {
```

Comment:
this was causing us to consider unpublished tracks as valid, breaking unpublished detection mechanism.